src/test/scala/edu/ie3/simona/model/participant/ChpModelSpec.scala

```diff
@@ -236,29 +238,32 @@ class ChpModelSpec
     "Check storage level after calculating next state with #chpState and heat demand #heatDemand kWh:" in {
       val testCases = Table(
         ("chpState", "storageLvl", "heatDemand", "expectedStoredEnergy"),
-        (chpStateNotRunning, 90, 0, 1035), // tests case (false, false, true)
+        (chpStateNotRunning, 70, 0, 805), // tests case (false, false, true)
         (
           chpStateNotRunning,
-          90,
+          70,
+          // todo check this
           8 * 115,
-          230,
+          0,
         ), // tests case (false, true, false)
-        (chpStateNotRunning, 90, 10, 1025), // tests case (false, true, true)
-        (chpStateRunning, 90, 0, 1135), // tests case (true, false, true)
-        (chpStateRunning, 90, 8 * 115, 230), // tests case (true, true, false)
-        (chpStateRunning, 90, 10, 1125), // tests case (true, true, true)
-        (
-          chpStateRunning,
-          90,
+        (chpStateNotRunning, 70, 10, 795), // tests case (false, true, true)
+        // todo check this, why 1135?
+        (chpStateRunning, 70, 0, 905), // tests case (true, false, true)
+        (chpStateRunning, 70, 8 * 115, 0), // tests case (true, true, false)
+        (chpStateRunning, 70, 10, 895), // tests case (true, true, true)
+        // todo, why not +100?
```

Comment:
```suggestion
```
demand is 806 kWh, chp will do 100 kWh, remaining demand is 706 kWh, storage contains 805 kWh so 99 will be in the end within the the storage. Or do I made any failure here?